manager/notifier.go

```diff
@@ -453,6 +455,30 @@ func (n *notifier) sendWebhookNotification(op notificationOp, config *pb.Webhook
 	return nil
 }
 
+func (n *notifier) sendAmazonSnsNotification(op notificationOp, config *pb.AmazonSnsConfig, a *Alert) error {
+	snsAPI := sns.New(nil)
+
+	status := ""
+	switch op {
+	case notificationOpTrigger:
+		status = "firing"
+	case notificationOpResolve:
+		status = "resolved"
+	}
+
+	params := &sns.PublishInput{
+		Message: aws.String(fmt.Sprintf("%s -- %s", a.Description, status)),
+		MessageStructure: aws.String("string"),
+		Subject:          aws.String(fmt.Sprintf("%s -- %s", a.Summary, status)),
+		TopicARN:         aws.String(config.GetTopicArn()),
+	}
+	_, err := snsAPI.Publish(params)
+	if err != nil {
```

Comment:
Seems like this line and the ones below can now simply be summarized as `return err`.